x/ibc/testing/solomachine.go

```diff
@@ -31,7 +31,7 @@ type Solomachine struct {
 // NewSolomachine returns a new solomachine instance with a generated private/public
 // key pair and a sequence starting at 1.
 func NewSolomachine(t *testing.T, cdc codec.BinaryMarshaler, clientID, diversifier string) *Solomachine {
-	privKey := ed25519.GenPrivKey()
+	privKey := secp256k1.GenPrivKey()
```

Comment:
I switched to secp256k1 to make some tests pass, but can't gauge of the impact it has. Is this all right?